### ISSUE TITLE
Problem downloading the test runner

### ISSUE THREAD

**Issue by gurghet** [zio/zio-intellij#266]
The following error(s) occurred while downloading the ZIO Test runner files:

```
Download error: zio-test-intellij_2.13:1.0.8"
Cause:
java.lang.RuntimeException: unresolved dependency: dev.zio#zio-test-intellij_2.13;1.0.8: not found
```
### Additional information:
<details>

ZIO plugin version: 2021.1.6.0
Scala plugin version: 2021.1.20
ZIO version(s): 1.0.7, 1.0.8
Scala version(s): 2.13.5, 2.13.5
IntelliJ version: IntelliJ IDEA 2021.1.1
Build #IU-211.7142.45
Java version: 11.0.10+9-b1341.41
Operating System: x86_64 Mac OS X (10.15.7, x86_64)
JVM version: Dynamic Code Evolution 64-Bit Server VM JetBrains s.r.o.
</details>

**Comment by hmemcpy** [comment]
Oof, that's right, new version... I'll take care of it shortly! :) Thanks!

**Comment by kpbochenek** [comment]
how is it going? :smiley: 

Not sure if this should be already fixed, but I am still getting:
```
Unable to download one or more required files. Please try again, or report this issue on GitHub, if the problem persists.
				Try again
				Report on GitHub (opens a browser)
```

(eap intellij + nightly scala + zio 1.0.8 + scala3)

**Comment by hmemcpy** [comment]
Oh hey, sorry about this. It should be fixed by the latest plugin update, but I've now looked at the error details again - seems you have two zio version references in your project (1.0.7 and 1.0.8). Maybe it confuses the downloader :( can you check that there's only a reference to one zio version in your sbt?

**Comment by kpbochenek** [comment]
build.sbt:
```
name := "scala3"
version := "0.1.0"
scalaVersion := "3.0.0"

val zioVersion = "1.0.8"

libraryDependencies ++= Seq(
  "org.typelevel" %% "cats-effect" % "3.1.1",
  "dev.zio" %% "zio" % zioVersion,
  "dev.zio" %% "zio-test" % zioVersion % Test,
  "dev.zio" %% "zio-test-sbt" % zioVersion % Test,
)

testFrameworks += new TestFramework("zio.test.sbt.ZTestFramework")

```

will check transitive deps but all looks correct

**Comment by kpbochenek** [comment]
Yup with `dependencyTree` it only points to zio 1.0.8 :( :

```
[info] scala3:scala3_3:0.1.0
[info]   +-dev.zio:zio_3:1.0.8
[info]   | +-dev.zio:izumi-reflect_3:1.1.2
[info]   | | +-dev.zio:izumi-reflect-thirdparty-boopickle-shaded_3:1.1.2
[info]   | | | +-org.scala-lang:scala3-library_3:3.0.0 [S]
[info]   | | |
[info]   | | +-org.scala-lang:scala3-library_3:3.0.0 [S]
[info]   | |
...
```

(tested sbt 1.5.1 and also checked on latest 1.5.2)

**Comment by hmemcpy** [comment]
Oof. Thank for the update, will keep testing. Stay tuned!

**Comment by kpbochenek** [comment]
Also not sure if this is related but I noticed Intellij cannot handle zio-test:1.0.8 for scala3 because of some weird errors.
(no completion, jumping to declarations, highlighting works)

They kind of point towards compiler, I noticed similar issue here and added my 3 cents:
https://github.com/lampepfl/dotty/issues/12482

**Comment by hmemcpy** [comment]
Hmm, I found the bug... it turns out that the dependency was published on maven as `zio-test-intellij-3` and not `zio-test-intellij-3.0.0` as the previous builds were... I wonder why that is. Either way, I'm probably handling the version juggling incorrectly, so I'll fix that in the plugin. Thanks again for reporting!

**Comment by hmemcpy** [comment]
So I made a hack^H^H^H^H fix for the issue. Turns out, the release version of Scala 3 treats library names differently... (prerelease versions were using e.g. `zio-test-intellij:3.0.0`, but now it's just `zio-test-intellij:3` (see [maven](https://repo1.maven.org/maven2/dev/zio/) for example) I have no idea why. So I fixed it for now, at least it can download the runner. Let's see how it breaks in the future :(

The fix will be available on the early access channel, I'll make a proper release in a few days... Thanks again for reporting!
